README.md

```diff
@@ -6,7 +6,9 @@
  Below are the sections contained in this project;
 - Toolbar
 - Headline section (right after the header).
-The layout was done using css flexbox in all 2 sections.
+- Works
+- About myself
+The layout was done using css flexbox and css grid, button interactions implemented.
```

Comment:
- [ ] your README looks good but it lacks one important section which is the `Getting started` section. This is where you state the procedures to clone and run your project locally 😃. This helps your fellow developers to know how to use your project 👍 so please add it there